pinecone/test_suite.go

```diff
@@ -54,7 +54,6 @@ func (ts *IntegrationTests) SetupSuite() {
 	for i, v := range vectors {
 		vectorIds[i] = v.Id
 	}
-	ts.vectorIds = append(ts.vectorIds, vectorIds...)
```

Comment:
This was being done twice. I think it's better to keep the actual appending where we upsert successfully. It's inside the `upsertVectors` helper function.